sickbeard/name_parser/parser.py

```diff
@@ -570,6 +575,10 @@ def is_anime(self):
             return True
         return False
 
+    @property
```

Comment:
There is not really a reason to create a property for this. i think you can just choose what you want to use. proper or is_proper. My preference is: is_proper, as that indicates, it is a bool.